hastexo/hastexo.py

```diff
@@ -273,6 +284,7 @@ def launch_or_resume_user_stack(self, sync = False):
         else:
             result = task.apply_async(args=args, kwargs=kwargs, expires=60)
             self.user_stack_launch_id = result.id
+            self.user_stack_launch_timestamp = datetime.datetime.now(pytz.utc)
```

Comment:
That's Python for ya:  http://stackoverflow.com/questions/796008/cant-subtract-offset-naive-and-offset-aware-datetimes

The XBlock DateTime uses the pytz.utc trick, so we have to, as well.